content/lessons/chapter-3/coop-3.tsx

```diff
@@ -196,6 +197,7 @@ export default function Coop3({ lang }) {
               key={i}
               username={profile.username}
               avatar={profile.avatar}
+              avatarSize={isSmallScreen ? 50 : 100}
```

Comment:
shouldn't the size be a 75? from the previous class i have seen min-w-[75px]

Reply:
This is on a challenge page the 75px icons are on intro pages

![image](https://github.com/saving-satoshi/saving-satoshi/assets/108441023/4ea2f2f1-da42-4c71-ac08-d420623c1a84)

vs

![image](https://github.com/saving-satoshi/saving-satoshi/assets/108441023/6c0eaa21-e30e-4028-b3cd-06a334f45376)

Reply:
oh yeah right. i remember that